x-pack/solutions/security/test/cloud_security_posture_api/routes/graph.ts

```diff
@@ -694,6 +696,205 @@ export default function (providerContext: FtrProviderContext) {
         expect(response.body).to.have.property('edges').length(2);
         expect(response.body).not.to.have.property('messages');
       });
+
+      describe('Asset inventory enabled', () => {
+        before(async () => {
+          // enable asset inventory feature flag
```

Comment:
_nit_: no need to comment trivial actions. the code is pretty descriptive IMO

ref: https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx#use-slashes-for-comments